### ISSUE TITLE
It should be possible to show a link to accessibility statement in the footer

### ISSUE THREAD

**Issue by wfenrich** [CeON/fairchive#1277]
If the page exists, we may like to put a link to it in the footer, after the "Terms of use" link.

**Comment by wfenrich** [comment]
Already made in https://github.com/CeON/dataverse/issues/863
